test/functional/ui/screen.lua

```diff
@@ -205,6 +205,8 @@ end
 
 function Screen:try_resize(columns, rows)
   uimeths.try_resize(columns, rows)
+  -- Give ourselves a chance to _handle_resize
+  self:sleep(0.1)
```

Comment:
This is sleeping 0.1ms, unless the input parameter for sleep is misnamed. Basically, give a chance for other stuff to run, then continue.